### ISSUE TITLE
Eve generate copy and remove problem.

### ISSUE THREAD

**Issue by cjzamora** [Sterling-Technologies/evejs#47]
Copy and Remove problem with eve generate (for linux), sometimes it's just throw out errors when trying to copy or remove a file while watching (specially for multiple files).

**Comment by cblanquera** [comment]
Yep i blame chokidar. Eventually i'll come around and make my own watcher, but this is what we got to work with. Before doing a mass operation, it is recommended to shut down the eve watcher then start it when complete and run `eve deploy`

**Comment by cblanquera** [comment]
I'll leave this open for now.
